server/src/main/java/com/linecorp/centraldogma/server/internal/api/TokenService.java

```diff
@@ -83,10 +84,15 @@ public TokenService(ProjectManager projectManager, CommandExecutor executor,
      * <p>Returns the list of the tokens generated before.
      */
     @Get("/tokens")
-    public CompletionStage<Collection<Token>> listTokens() {
-        return mds.getTokens()
-                  .thenApply(Tokens::withoutSecret)
-                  .thenApply(tokens -> tokens.appIds().values());
+    public CompletionStage<Collection<Token>> listTokens(@RequestObject User loginUser) {
+        if (loginUser.isAdmin()) {
+            return mds.getTokens()
+                      .thenApply(tokens -> tokens.appIds().values());
+        } else {
+            return mds.getTokens()
+                      .thenApply(Tokens::withoutSecret)
```

Comment:
No need to hide admin token here?

Reply:
I think we don't need to do it because we hide every `secret` here.  :-)

Reply:
Got it!!